.atomist/mocha/SimpleTest.ts

```diff
@@ -1,10 +0,0 @@
-import "mocha";
-import assert = require("power-assert");
-
-describe("default test", () => {
-
-    it("should be true", () => {
-        assert(true === true);
-    });
-
-});
```

Comment:
This will get re-added next time atomist:rug-rugs:UpdateSupportFiles is run on this project.